### PR TITLE
Finish Form contact

### PR DESCRIPTION
**Pull request by Chaw-Ma-Ma-Win**
### Finishing  Form Contact

- Making form with Html
- Styling with CSS

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -142,7 +144,7 @@ <h2 id="footer-head" class="footer-heading">Contact me</h2>
                 </li>
                 <button type="submit" id="submit"  class="submitform">Get in touch</button>
                </ul>
-            </form>
+            </form> -->
```
Comment (Abenezer-Tilahun) [inline index.html:147]: - [ ] Good Job for implementing the Project requirement But You commented out the `Form` section So the Form is not displaying as shown the screenshot  Kindly Fix the issues.
contact

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Chaw-Ma-Ma-Win @denigit5   , 

Great job for implemented  the some of the Project Requirement , but there are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻
✔️ No Linter errors
✔️ Correct Git Hub-Flow
✔️ Good commit messages
✔️ Professional README

almost

## Required Changes ♻️
_Check the comments below the review._


_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html
Comment (AbelG101) [inline index.html:1]: - [ ] There seems to be a small discrepancy in your design of the message field (textarea) as I noticed It's using the default browser font for that element and you're also missing padding around it, please refer to the below Figma design 👇
- Figma design:
  image
- Your design:
  image

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @denigit5 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Form is properly validated ✔
- Formspree is correctly integrated ✔
- Correct use of GitHub Flow ✔
- Proper branch name ✔
- All linter checks are passing ✔

 ## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alusp — APPROVED** [review]
Hi @denigit5, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congratulations

## Highlights

- [ ] Use `formspree.io` :heavy_check_mark: 
- [ ] Use Validations Correctly :heavy_check_mark: 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me  @Alusp in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**